ProgramFiles/Utilities/curvature_mode_toolbox/illustrate_backbone_shapespace.m

```diff
@@ -14,18 +28,29 @@ function illustrate_backbone_shapespace(curvdef,paramvalues,orientation)
     error('Unsupported curvature definition')
 end
 
+% Specify plot number as 171 unless specified otherwise
+if ~exist('plotnum','var')
+    plotnum = 171; % 171 was the hardcoded default before I modified this
+end
+
 % Get the gradient of parameter values in the first two dimensions
 [~,grid_spacing_x] = gradient(paramvalues{1});
 [grid_spacing_y,~] = gradient(paramvalues{2});
 
-fh = figure(171);
+% Create the figure
+fh = figure(plotnum);
 close(fh);
-fh = figure(171);
-
+fh = figure(plotnum);
+set(gcf,'name',orientation);
```

Comment:
this should be

set(fh,'name',orientation)

since we know the handle for the figure already, and coding it this way avoids a race condition with any other process deciding that it's going to bring a different figure to the top

Reply:
Done!